### DISCUSSION TITLE
ML4W Dotfiles 2.9.5RC1 is ready for testing

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [mylinuxforwork/dotfiles#103] [Announcements]
DO YOU WANT TO TEST ALREADY TODAY THE UPCOMING ML4W Dotfiles 2.9.5?

The Release Candidate 1 is now available on the Rolling Release (main branch). Please report your findings as an issue on GitHub. https://github.com/mylinuxforwork/dotfiles

Changelog:
https://github.com/mylinuxforwork/dotfiles/blob/main/CHANGELOG.md

Also the Wiki has been updated:
https://github.com/mylinuxforwork/dotfiles/wiki

Thanks in advance to all testers.

**Comment by FurtherSecrets24680** [comment]
**Currently testing the dotfiles...**

After installing the dotfiles, I was very confused about why my keybindings weren't working, also the wallpaper, waybar etc. wasn't showing.

Then I thought the file structure was reconstructed, then I viewed my keybindings and autostart... found that it was using the old config (because I replaced it with the previous one as it contained my custom config). So, I changed the path to the programs as it is in the new config. Then it started working perfectly. Should've known it earlier.

I also have a suggestion, the browser icon in the waybar, is just firefox. Everyone doesn't use firefox, so it'd be better to add a universal browser icon (globe?,)  there, or during installation, it'll ask the user to choose their own browser, and based on it, the waybar browser icon will be chosen. Also, replace the "firefox" command (which is run by clicking the browser icon) with browser.sh in the dotfiles.

Edit: Screenshot - 
![image](https://github.com/user-attachments/assets/4bd16a46-d16d-4753-947f-494a002495f8)

**Reply by mylinuxforwork** [reply]
Thank you so much. I will add a special note in the Readme about the new file structure.

The waybar-quicklinks.json is now using the browser.sh. but you're right, the Firefox icon is used. 

The current idea is that users can customize the waybar-quicklinks.json and add new icons individually. 

The waybar-quicklinks.json can be restored during the update.

**Comment by ekremx25** [comment]
Fn f1 not cut volume..

**Reply by mylinuxforwork** [reply]
I guess it depends on the keyboard variant.
